pkg/datastore/sql_struct.go

```diff
@@ -44,6 +44,7 @@ const (
 
 // Maps DB table name to booleans showing if the table supports multitenancy and revisioning.
 var schemaMap = map[string]map[string]bool{}
```

Comment:
I propose you replace `map[string]map[string]bool{}` with `map[string]map[string]struct{}`\
This will make this map easier to understand